### PR TITLE
Galician language update.

### PR DESCRIPTION
**Pull request by rafacouto**
Update of Galician language 2016-12-05

### CODE REVIEW

File: Marlin/language_gl.h [hunk 1]
```diff
@@ -182,6 +185,8 @@
 #define MSG_INFO_EXTRUDERS                  "Extrusores"
 #define MSG_INFO_BAUDRATE                   "Baudios"
 #define MSG_INFO_PROTOCOL                   "Protocolo"
+#define MSG_LIGHTS_ON                       "Acender luz LCD"
+#define MSG_LIGHTS_OFF                      "Apagar luz LCD"
```
Comment (Blue-Marlin) [inline Marlin/language_gl.h]: This has nothing to do with the LCD. It's a lamp somewhere in the printer.

Comment (thinkyhead) [reply Marlin/language_gl.h]: Podemos usar "**`Acender a luz`**" e "**`Apagar as luces`**"?

Comment (rafacouto) [reply Marlin/language_gl.h]: @Blue-Marlin You are right: I have reviewed in code, those strings are in ultralcd.cpp lcd_main_menu() and I got confused. 
@thinkyhead I new PR is comming.
Thank you 👍

**Comment by thinkyhead** [comment]
Squashed and merged at #5386

**Pull request closed without merging**